CONTRIBUTING.md

```diff
@@ -1225,7 +1225,13 @@ $ yarn test
 ## Changing Cloud Assembly Schema
 
 If you plan on making changes to the `cloud-assembly-schema` package, make sure you familiarize yourself with
-its own [contribution guide](./packages/@aws-cdk/cloud-assembly-schema/CONTRIBUTING.md)
+its own [contribution guide](https://github.com/cdklabs/cloud-assembly-schema/blob/main/CONTRIBUTING.md).
+
+The team will not actively monitor the [cloud-assembly-schema](https://github.com/cdklabs/cloud-assembly-schema)
+repository for contributions. If you have a PR that needs to modify `cloud-assembly-schema`, open both PRs and
+ensure that you link the `cloud-assembly-schema` PR in the one opened against this repository.
```

Comment:
i don't understand what PR would be opened against aws-cdk's cloud-assembly-schema given that its just a shell for the external version.